jest.ts.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+  globals: {
+    'ts-jest': {
+      babelConfig: 'babel.config.js',
+      tsConfig: './examples/using-typescript/tsconfig.json',
+    },
+  },
+  setupFilesAfterEnv: ['./scripts/testSetup.js'],
+  testURL: 'http://react-easy-state.com',
+  testRegex: '\\.test\\.tsx?$',
+  collectCoverage: true,
+  preset: 'ts-jest',
+  testEnvironment: 'jsdom',
+  coverageReporters: ['text'],
+  transform: {
+    '^.+\\.jsx?$': 'babel-jest', // Adding this line solved the issue
```

Comment:
What issue? Can you clarify in the comment (a link to the issues would be best).

Reply:
Sorry that looks to me as a leftover from an issue thread somewhere online, but the thing is that jest needs to know how to transform js files.